src/Tools/dotnet-user-secrets/src/Internal/EditCommand.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.Extensions.SecretManager.Tools.Internal
+{
+    internal class EditCommand : ICommand
+    {
+        public void Execute(CommandContext context)
+        {
+            Process.Start(context.SecretStore.SecretsFilePath);
```

Comment:
Does this work in linux? According to https://stackoverflow.com/questions/13627767/open-file-in-default-editor-from-bash, you might have to do more work.